File: src/main/java/pl/yalgrin/playnite/simplesync/mapper/objects/GenreMapper.java
package pl.yalgrin.playnite.simplesync.mapper.objects;

import org.springframework.stereotype.Component;
import pl.yalgrin.playnite.simplesync.library.domain.Genre;
import pl.yalgrin.playnite.simplesync.library.dto.GenreDTO;

@Component
public class GenreMapper extends AbstractObjectMapper<Genre, GenreDTO> {

    @Override
    protected GenreDTO createDTO() {
        return new GenreDTO();
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/mapper/GenreMapper.kt
package pl.yalgrin.playnite.simplesync.library.mapper

import org.springframework.stereotype.Component
import pl.yalgrin.playnite.simplesync.library.domain.Genre
import pl.yalgrin.playnite.simplesync.library.dto.GenreDTO

@Component
class GenreMapper : LibraryObjectMapperImpl<Genre, GenreDTO>() {
    override fun createDTO(): GenreDTO = GenreDTO()
}